robots/pkg/flakefinder/flakefinder.go

```diff
@@ -226,11 +227,17 @@ func GetReportBaseData(ctx context.Context, c *github.Client, client *storage.Cl
 		reports = append(reports, r...)
 	}
 
+	batchJobResults, err := FindUnitTestFilesForBatchJobs(ctx, client, BucketName, regexp.MustCompile("pull-kubevirt-e2e-k8s-.*"), prs, startOfReport, endOfReport)
```

Comment:
Thanks, `kind` and `windows` jobs are out of our scope?
Because I see also
- pull-kubevirt-e2e-windows2016
- pull-kubevirt-e2e-kind-1.23-vgpu-nonroot
which will be excluded from the regex

Reply:
Oh, you are absolutely right! :blush: 
I'll fix this.

Reply:
@fossedihelm according to what you suggested, I've changed the regex. I've also extracted it into a flag, which I think makes more sense than burying it deep inside the code base.

Please see commit https://github.com/kubevirt/project-infra/pull/2350/commits/a369b703e7feba4ecfcd4000b3ce80ac6b73facb#diff-396be98f4f2cced59b53a687925ff277f72c326fd817eccb71a34303b1c6536aR56